.github/workflows/source_firestore.yml

```diff
@@ -50,6 +50,17 @@ jobs:
           args: "--config ../../.golangci.yml"
           skip-pkg-cache: true
           skip-build-cache: true
+      - name: Setup CloudQuery
+        if: github.event_name == 'pull_request'
+        uses: cloudquery/setup-cloudquery@v3
+        with:
+          version: v5.1.2
```

Comment:
```suggestion
```

not required here

Reply:
Done [here](https://github.com/cloudquery/cloudquery/pull/16380/commits/02d6bdfcdf1ddef7eb10f0fd0ff7ab7d20953ccf)